src/index.tsx

```diff
@@ -12,7 +12,7 @@ const style = {
     transition: "background 0.25s ease-in-out",
     backgroundPosition: "center",
```

Comment:
Hi @jbaxleyiii, I'm finally back from my month-long honeymoon vacation. Looks like you were able to merge this. Were you able to drop the backgroundPosition prop before/after the merge?